### ISSUE TITLE
How to see train score and test scores on trained data

### ISSUE THREAD

**Issue by sathishkumar369** [pycaret/pycaret#676]
Hi team,

Could you please clarify me.
If compare models or create model gives the train score. so how could I see the test score with metrics. I'm confusing, whether create model is giving train score or test score.
Because I just wanted to know, whether my model is overfitted or underfitted. 
In learning curve I was able to see train and test score. Is it possible to get metrics with data frame format for training and testing accuracy for the selected model.

Thanks

**Comment by Yard1** [comment]
`create_model()` is returning cross-validation scores on the train set. In order to obtain scores and predictions on the test set, use `predict_model()`.

**Comment by sathishkumar369** [comment]
thanks that helped

**Comment by gm039** [comment]
Hi,
I understand that the create_model() returns cross-validation scores on the train set averaged over n_iter.
Can I get the cross-validation scores on each iteration?
